soweego/importer/imdb_dump_extractor.py

```diff
@@ -118,17 +118,33 @@ def extract_and_populate(self, dump_file_paths: List[str], resolve: bool) -> Non
             movie_entity = imdb_entity.ImdbMovieEntity()
             movie_entity.catalog_id = movie_info.get('tconst')
             movie_entity.title_type = movie_info.get('titleType')
-            movie_entity.primary_title = movie_info.get('primaryTitle')
-            movie_entity.original_title = movie_info.get('originalTitle')
+            movie_entity.name = movie_info.get('primaryTitle')
+            movie_entity.name_tokens = ' '.join(text_utils.tokenize(movie_info.get('primaryTitle')))
             movie_entity.is_adult = True if movie_info.get(
                 'isAdult') == '1' else False
-            movie_entity.start_year = movie_info.get('startYear')
-            movie_entity.end_year = movie_info.get('endYear')
+            try:
+                movie_entity.born = datetime.date(year=int(movie_info.get('startYear')), month=1, day=1)
+                movie_entity.born_precision = 9
+            except:
+                pass
+            try:
+                movie_entity.end_year = datetime.date(year=int(movie_info.get('endYear')), month=1, day=1)
```

Comment:
```suggestion
                movie_entity.died = datetime.date(year=int(movie_info.get('endYear')), month=1, day=1)
```
The `end_year` field was removed in the DB entity model, so I assume it wasn't updated here.

Reply:
You're right!